Appraisals

```diff
@@ -14,6 +14,6 @@ appraise "rails-4.2.0" do
 end
 
 
-appraise "rails-5.0.0" do
-  gem "rails", "5.0.0"
+appraise "rails-5.0.3" do
```

Comment:
Bumps up version of rails from 5.0.0 to 5.0.3 to include a bug fix in the minitest plugin rails/rails#29022. This only prevents the test suit from crashing.

I won't submit a PR to the original repo because this other PR will include the fix inossidabile#238